parameter/ParameterMgr.cpp

```diff
@@ -93,15 +90,27 @@
 #include <sstream>
 #include <algorithm>
 #include <ctype.h>
-#include <memory>
 
 #define base CElement
 
+/** Private macro helper to declare a new context
+ *
+ * Context declaration always need logger and logging prefix to be
+ * passed as parameters.
+ * This macro aims to avoid this boring notation.
+ * This macro should be called only once in a scope.
```

Comment:
as a new scope is defined I suppose it's understandable (it was what I meant). Maybe we can precise that variable shadowing is authorized..